uktrade/data-hub-frontend
# Add companies list to homepage dashboard

## Description of change
This enables users to see the companies they follow on the homepage. To add companies to their list they have to click "add to list" on a company page which was story #2070 (feature flagged "add_company_to_list").

## Test instructions
You will need to turn the feature flag "add_company_to_list" on so you can add companies to your list (see story #2070). Once this is enabled just add the companies from the company page then view the list on the home page.
 
## Screenshots
### Before
![Screenshot 2019-08-22 at 19 31 36](https://user-images.githubusercontent.com/10154302/63540174-8355ce80-c513-11e9-9b61-d83e3a55ba26.png)

### After 
![Screenshot 2019-08-22 at 19 31 42](https://user-images.githubusercontent.com/10154302/63540193-8cdf3680-c513-11e9-8cc6-184f0ab26505.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
